modules/ROOT/partials/configuration/advcode.adoc

```diff
@@ -0,0 +1,24 @@
+[[advcode_inline]]
+== `+advcode_inline+`
+
+include::partial$misc/admon-requires-6.3v.adoc[]
+
+As part of the {productname} 6.3 release, the {pluginname} plugin includes a new option, `advcode_inline`, that allows users to open the Advanced Code Editor within {productname}’s existing editor space instead of been displayed in a seperate dialog box.
+
+Using `advcode` inside {productname} means users on mobile devices or with otherwise narrow screens do not require a `ToggleView` button, as `advcode` forces the editor to open within the existing editor space in inline mode.
```

Comment:
I'm not sure if `ToggleView` is a button. The only things we define for the toolbar / menu in advcode is `code` (https://github.com/tinymce/advcode/blob/main/src/main/ts/ui/Controls.ts)